shakenfist/virt.py

```diff
@@ -170,47 +178,203 @@ def from_db(uuid):
 
         return Instance(static_values)
 
+    # NOTE(mikal): I wish we could generate this through introspection or
+    # something, this is ugly.
+    @property
+    def uuid(self):
+        return self.__uuid
+
+    @uuid.setter
+    def uuid(self, _):
+        raise exceptions.NotImplementedException(
+            'You cannot set a static variable')
+
+    @uuid.deleter
+    def uuid(self):
+        raise exceptions.NotImplementedException(
+            'You cannot delete a static variable')
```

Comment:
I think you could reduce code by not defining the `setter` and `deleter`. If they are called I think you get an `AttributeError` which is probably more pythonic.

Reply:
Oh interesting. I will test this when I am not in a meeting.

Reply:
And you're right. Fixed.